crates/bin/pindexer/src/dex/dex.sql

```diff
@@ -9,13 +9,17 @@
 -- that given an `penumbra_asset::asset::Id`, we always know exactly how to filter
 -- tables, rather than needing to do a join with another table.
 
-CREATE DOMAIN IF NOT EXISTS Amount AS NUMERIC(39, 0) NOT NULL;
 
+DROP TYPE IF EXISTS Value CASCADE;
+DROP DOMAIN IF EXISTS Amount;
+
+CREATE DOMAIN Amount AS NUMERIC(39, 0) NOT NULL;
 CREATE TYPE Value AS (
   amount Amount,
-  asset BYTEA NOT NULL
+  asset BYTEA
```

Comment:
If the `NOT NULL` constraint intended here previously is critical, this could be refactored with something like `CREATE DOMAIN Asset AS BYTEA NOT NULL;` but that might cause translation issues & require a bit more work that this